qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -259,3 +264,9 @@ def _incremental_search(self, text):
 
         if self.prefix() in ['/', '?']:
             self.got_cmd[str].emit(search_prefixes[text[0]] + text[1:])
+
+    @pyqtSlot(str)
+    def _exit_prefix(self, text):
+        if config.val.completion.exit_prefix and (not text or text[0] not in self.prefixes):
```

Comment:
How can `text[0] not in self.prefixes` happen? Wouldn't it always be completely empty?

Reply:
In case the user only deletes the prefix and leaves the rest of the command.